com.sayonara_player.Sayonara.json

```diff
@@ -0,0 +1,186 @@
+            "sources": [
+                {
+                    "commit": "e816c6cd73c9e0676828c9e227a049ebad3d019f",
+                    "tag": "1.18.3",
+                    "type": "git",
+                    "url": "https://gitlab.freedesktop.org/gstreamer/gst-plugins-good.git"
+                }
+            ]
+        },
+        {
+            "config-opts": [
+                "--buildtype=release",
+                "-Ddoc=disabled",
+                "-Dexamples=disabled",
+                "-Dtests=disabled",
+                "-Dnls=disabled",
+                "-Dgobject-cast-checks=disabled",
+                "-Dglib-asserts=disabled",
+                "-Dglib-checks=disabled",
+                "-Dextra-checks=disabled",
+                "-Dflite=disabled",
+                "-Dsoundtouch=enabled"
+            ],
+            "buildsystem": "meson",
+            "name": "gstreamer-plugins-bad",
+            "sources": [
+                {
+                    "commit": "382e373d9be363f1e21b12990a4d12f1ecb6df41",
+                    "tag": "1.18.3",
+                    "type": "git",
+                    "url": "https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad.git"
+                }
+            ]
+        },
+        {
+            "config-opts": [
+                "--buildtype=release",
+                "-Ddoc=disabled",
+                "-Dnls=disabled",
+                "-Dtests=disabled",
+                "-Dgobject-cast-checks=disabled",
+                "-Dglib-asserts=disabled",
+                "-Dglib-checks=disabled"
+            ],
+            "buildsystem": "meson",
+            "name": "gstreamer-plugins-ugly",
+            "sources": [
+                {
+                    "commit": "ad60e5463f591caa1c8470f180fd0ef8fce4a802",
+                    "tag": "1.18.3",
+                    "type": "git",
+                    "url": "https://gitlab.freedesktop.org/gstreamer/gst-plugins-ugly.git"
+                }
+            ]
+        },
+        {
+            "config-opts": [
+                "-DBUILD_SHARED_LIBS=ON"
+            ],
+            "name": "taglib",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://taglib.github.io/releases/taglib-1.11.1.tar.gz",
+                    "sha256": "b6d1a5a610aae6ff39d93de5efd0fdc787aa9e9dc1e7026fa4c961b26563526b"
+                }
+            ]
+        },
+        {
+            "name": "sayonara",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_INSTALL_PREFIX=/app",
+                "-DCMAKE_BUILD_TYPE=Release"
+            ],
+            "sources": [
+                {
+                    "commit": "18e63ab31da929ebfa4d7f21791039997a567b49",
+                    "type": "git",
+                    "url": "https://gitlab.com/luciocarreras/sayonara-player.git",
```

Comment:
It's not a blocker, but consider adding https://github.com/flathub/flatpak-external-data-checker/ for automatic release monitoring.

Reply:
Thanks for the hint!